flasksrc/seed_data/toynet_values.sql

```diff
@@ -20,3 +20,4 @@ INSERT INTO toynet_value_inspirations(value_id, organization, quote) VALUES(5003
 
 INSERT INTO toynet_values(id, name) VALUES(5004, "Loyalty");
 INSERT INTO toynet_value_inspirations(value_id, organization, quote) VALUES(5004, "U.S. Army", "Bear true faith and allegiance to the U.S. Constitution, the Army, your unit and other Soldiers. Bearing true faith and allegiance is a matter of believing in and devoting yourself to something or someone … By wearing the uniform of the U.S. Army you are expressing your loyalty. And by doing your share, you show your loyalty to your unit.");
+
```

Comment:
Since you are addressing the other item, let's delete this new line so others can reference this PR later!